docs/4.0/components/modal.md

```diff
@@ -67,8 +60,8 @@ Below is a _static_ modal example (meaning its `position` and `display` have bee
         <p>Modal body text goes here.</p>
       </div>
       <div class="modal-footer">
-        <button type="button" class="btn btn-primary">Save changes</button>
```

Comment:
Static modal example's highlight html didn't match the inline-html (close button should come first)